File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/04.For-Loop/02-exercise/07.trekkingMania.js
function trekkingMania(input) {

    const numberOfGrups = Number(input[0]);
    let index = 1;

    let climbingMusala = 0;
    let climbingMontBlanc = 0;
    let climbingKilimanjaro = 0;
    let climbingK2 = 0;
    let climbingEverest = 0;

    for (let i = 1; i <= numberOfGrups; i++) {
        let peopleInCurrentGrup = Number(input[index++]);
        if (peopleInCurrentGrup > 40) {
            climbingEverest += peopleInCurrentGrup;
        } else if (peopleInCurrentGrup > 25) {
            climbingK2 += peopleInCurrentGrup;
        } else if (peopleInCurrentGrup > 12) {
            climbingKilimanjaro += peopleInCurrentGrup;
        } else if (peopleInCurrentGrup > 5) {
            climbingMontBlanc += peopleInCurrentGrup;
        } else {
            climbingMusala += peopleInCurrentGrup;
        }
    }

    const allClimbers = climbingMusala + climbingMontBlanc + climbingKilimanjaro + climbingK2 + climbingEverest;
    const climbingMusalaPercentage = climbingMusala / allClimbers * 100;
    const climbingMontBlancPercentage = climbingMontBlanc / allClimbers * 100;
    const climbingKilimanjaroPercentage = climbingKilimanjaro / allClimbers * 100;
    const climbingK2Percentage = climbingK2 / allClimbers * 100;
    const climbingEverestPercentage = climbingEverest / allClimbers * 100;

    console.log(climbingMusalaPercentage.toFixed(2) + "%");
    console.log(climbingMontBlancPercentage.toFixed(2) + "%");
    console.log(climbingKilimanjaroPercentage.toFixed(2) + "%");
    console.log(climbingK2Percentage.toFixed(2) + "%");
    console.log(climbingEverestPercentage.toFixed(2) + "%");
}
trekkingMania(["10",
    "10",
    "5",
    "1",
    "100",
    "12",
    "26",
    "17",
    "37",
    "40",
    "78"]);

File: Python_Basics/Homeworks-And-Labs/04.for_loop/02.exercise/07.trekking_mania.py
number_of_groups = int(input())

climbing_musala = 0
climbing_mont_blanc = 0
climbing_kilimanjaro = 0
climbing_k2 = 0
climbing_everest = 0

for i in range(number_of_groups):
    people_in_current_group = int(input())
    if people_in_current_group > 40:
        climbing_everest += people_in_current_group
    elif people_in_current_group > 25:
        climbing_k2 += people_in_current_group
    elif people_in_current_group > 12:
        climbing_kilimanjaro += people_in_current_group
    elif people_in_current_group > 5:
        climbing_mont_blanc += people_in_current_group
    else:
        climbing_musala += people_in_current_group

all_climbers = climbing_musala + climbing_mont_blanc + climbing_kilimanjaro + climbing_k2 + climbing_everest
climbing_musala_percentage = climbing_musala / all_climbers * 100
climbing_mont_blanc_percentage = climbing_mont_blanc / all_climbers * 100
climbing_kilimanjaro_percentage = climbing_kilimanjaro / all_climbers * 100
climbing_k2_percentage = climbing_k2 / all_climbers * 100
climbing_everest_percentage = climbing_everest / all_climbers * 100

print(f'{climbing_musala_percentage:.2f}%')
print(f'{climbing_mont_blanc_percentage:.2f}%')
print(f'{climbing_kilimanjaro_percentage:.2f}%')
print(f'{climbing_k2_percentage:.2f}%')
print(f'{climbing_everest_percentage:.2f}%')
